Castle/Public/Castle.m

```diff
@@ -312,11 +327,17 @@ + (BOOL)isApplicationLifecycleTrackingEnabled
 
 - (void)setUserJwt:(nullable NSString *)userJwt
 {
+    if ((userJwt == nil && _userJwt == nil) || [userJwt isEqualToString:_userJwt]) {
+        return;
+    }
+    
```

Comment:
Could we simplify like this?

if ([userJwt isEqual:_userJwt]) {
    return;
}

Reply:
Need to check for nil before or it will crash